blocks/file-blocks/excalidraw/index.tsx

```diff
@@ -28,7 +35,9 @@ export default function (props: FileBlockProps) {
       console.error("Excalidraw is not loaded.");
       return;
     }
+    const newVersion = excalModule.getSceneVersion(elements);
     const serialized = excalModule.serializeAsJSON(elements, appState);
+    if (newVersion === version) return;
```

Comment:
could check the version before calling `serializeAsJSON`.

I think what you have fixes the initial load but for subsequent `handleChange` calls you should call `setVersion` here.

Reply:
Made an oopsies when transcribing over from `blocks` Will update. TY!